permits/models.py

```diff
@@ -427,7 +427,14 @@ class PermitRequest(models.Model):
         (PROLONGATION_STATUS_APPROVED, _("Approuvée")),
         (PROLONGATION_STATUS_REJECTED, _("Refusée")),
     )
-
+    shortname = models.CharField(
+        _("nom de la demande"),
+        max_length=32,
```

Comment:
In the story it's 64

Reply:
Toooo long after checking with @benschnell